tests/sentry/api/bases/test_organization.py

```diff
@@ -242,7 +244,7 @@ def test_staff(self, mock_is_active_staff):
     @mock.patch("sentry.api.permissions.is_active_staff", wraps=is_active_staff)
     def test_staff_passes_2FA(self, mock_is_active_staff):
         staff_user = self.create_user(is_staff=True)
-        request = self.make_request(user=staff_user, is_staff=True)
+        request = self.make_request(user=serialize_rpc_user(staff_user), is_staff=True)
```

Comment:
just wondering why this needs to be serialized?

Reply:
Hybrid cloud magic lol, see https://github.com/getsentry/sentry/pull/65490